routing_common/transit_types.hpp

```diff
@@ -296,12 +298,29 @@ class Transfer
   std::vector<TitleAnchor> m_titleAnchors;
 };
 
+class StopIdRanges
+{
+public:
+  StopIdRanges() = default;
+  explicit StopIdRanges(Ranges const & ids) : m_ids(ids) {}
+  bool operator==(StopIdRanges const & rhs) const { return m_ids == rhs.m_ids; }
```

Comment:
Please, add a blank line betwee ctor and operator==.

Reply:
Done for all classes at transit_types.